S10-packages/precompilation.t

```diff
@@ -10,9 +10,10 @@ my $precomp-ext    := $*VM.precomp-ext;
 my $precomp-target := $*VM.precomp-target;
 my @precomp-paths;
 
-my @precompiled = Test::Util::run( q:to"--END--").lines;
-    use lib 't/spec/packages';
+BEGIN my $lib-path = $?FILE.IO.parent(2).IO;
+my $package-lib-prefix = $lib-path.add('packages').absolute;
 
+my @precompiled = Test::Util::run( "use lib <{$package-lib-prefix}>;\n" ~ q:to"--END--").lines;
```

Comment:
Hmm, is there a universally safe way to do so? Otherwise will require passing in the lib another way.

fwiw you cannot install rakudo to a space with a path either, but better to program defensively here I agree.

Reply:
Maybe `"use lib $package-lib-prefix.perl();\n"` ?